.github/dependabot.yml

```diff
@@ -15,25 +15,27 @@ updates:
   labels:
     - 'dependencies'
 # maintaining v2
-- package_manager: "java:gradle"
-  target_branch: "release/2.x.x"
+- package-ecosystem: "java:gradle"
+  target-branch: "release/2.x.x"
   directory: "/"
-  update_schedule: "weekly"
-  default_labels:
+  schedule:
+    interval: "weekly"
+  labels:
     - "dependencies"
     - "release-2.x.x"
-  commit_message:
+  commit-message:
     prefix: "fix"
-    prefix_development: "chore"
-    include_scope: true
-- package_manager: "github_actions"
-  target_branch: "release/2.x.x"
+    prefix-development: "chore"
+    include: scope
+- package-ecosystem: "github-actions"
```

Comment:
Oh... did that work? Do we want updates for Github actions only for the release branch?

Reply:
That's covered with renovate for the main branch to group the updates: [Example](https://github.com/SDA-SE/sda-dropwizard-commons/pull/2020)